### ISSUE TITLE
Exercise 2.64

### ISSUE THREAD

**Issue by dandanua** [goropikari/SolutionForQuantumComputationAndQuantumInformation#4]
The actual solution is not too hard (though I also struggled a bit). 

Let $1 \le k \le m$ and denote 
$H = span\{\phi_1,...,\phi_m \}$,
$H_k = span\{\phi_1,...,\phi_{k-1}, \phi_{k+1},...,\phi_m \}$. 
Because of linear independence, subspace $H$ has dimension $m$ and $H_k$ has dimension $m-1$. Now denote $E_k = Proj(H) - Proj(H_k)$, where $Proj(H)$ denotes the (ortho)projection operator on the subspace $H$. Clearly, $E_k \ge 0$ and $E_k \neq 0$.
Also, denote $E_{m+1} = I - E_1-...-E_m$.  
Now calculate $\langle \phi_i|E_k|\phi_i \rangle = \langle \phi_i|Proj(H)|\phi_i \rangle - \langle \phi_i|Proj(H_k)|\phi_i \rangle = 1 - \langle \phi_i|Proj(H_k)|\phi_i \rangle = 0$ if $i \neq k$ and $>0$ if $i=k$. This is what we needed.

**Comment by goropikari** [comment]
> Also, denote $E_{m+1} = I - E_1-...-E_m$.

How do you ensure $E_{m+1}$ is non-negative?
I think it can be negative in this definition.

**Comment by dandanua** [comment]
Yes, it's not clear. Well, anyway, we can scale all $E_i, i <= m$, i.e. take $E_i = \alpha E_i$ for some sufficiently  small $\alpha > 0$.
